Kickstarter-iOS/Views/Controllers/CancelPledgeViewController.swift

```diff
@@ -122,6 +125,7 @@ final class CancelPledgeViewController: UIViewController {
 
     _ = self.cancellationReasonTextField
       |> cancellationReasonTextFieldStyle
+      |> UITextField.lens.accessibilityLabel %~ { _ in Strings.Cancellation_reason() }
       |> UITextField.lens.placeholder %~ { _ in Strings.Tell_us_why_optional() }
```

Comment:
Can we get rid of the lens and do ` |> \.accessibilityLabel %~ { _ in Strings.Cancellation_reason() }` and  ` |> \.placeholder %~ { _ in Strings.Tell_us_why_optional() }` here?